packages/client/components/TeamPrompt/TeamPromptResponseCard.tsx

```diff
@@ -86,7 +90,10 @@ const ResponseCardFooter = styled('div')({
 
 export const TeamMemberName = styled('h3')({
   padding: '0 8px',
-  margin: 0
+  margin: 0,
+  width: '90%',
```

Comment:
I think instead of hardcoding a number we can do the following

<img width="317" alt="image" src="https://user-images.githubusercontent.com/466991/214026507-6e682b30-0935-4cc5-be6a-20d44fcbca0a.png">


<img width="409" alt="image" src="https://user-images.githubusercontent.com/466991/214026452-cce973b2-4a03-4319-bd9b-d8788004fc4e.png">